app/migrations/0001_initial.py

```diff
@@ -0,0 +1,61 @@
+# Generated by Django 4.2.3 on 2023-07-25 08:25
+
+from django.conf import settings
+import django.contrib.auth.models
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
```

Comment:
please squash all these migrations into one migration, the best practice is to have only one migration per PR